include/glow/Backends/CompiledFunction.h

```diff
@@ -28,6 +30,7 @@ class CompiledFunction {
 
   /// Execute the network.
   virtual void execute() = 0;
+  virtual void execute(Context &ctx) = 0;
```

Comment:
can you flesh out comments on what ctx is etc in the Doxygen comment?

Reply:
Hmm, this doesn't necessarily need to hold this PR, but I am not sure I like the fact that the executer does the memory allocation. My concern is that it prevents us to reuse already allocated buffer from another run.
I.e., I would rather have the context have the location of the different placeholder at this point, instead of allocating them.
Anyhow, that's a more general design discussion, again no need to hold that PR, we would need the context to execute anyway.